src/oas2/transformers/params.ts

```diff
@@ -192,6 +192,8 @@ function buildSchemaForParameter(
   );
 
   if ('allowEmptyValue' in param && param.allowEmptyValue) {
+    delete schema.minLength;
```

Comment:
What is this fixing?

Reply:
Whoops, my bad — I thought I updated the PR text.

Basically if `allowEmptyValue` is true, the `minLenght` property is "unset" so that empty strings are allowed.